templates/zerver/help/using-zulip-for-a-class.md

```diff
@@ -46,20 +48,23 @@ might see the following topics in a stream where a lecture and the
 corresponding assignment are being discussed:
 
 * problem 2a clarification
-* question about slide 7
```

Comment:
I don't really agree with this change. Also, the current list is what we have on the screenshot on https://zulip.com/for/education/.